client/src/components/MainPage/TranslationsPage/DonateModal/VideoTranslation/index.jsx

```diff
@@ -1,12 +1,54 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import './style.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faVideo, faStop } from '@fortawesome/free-solid-svg-icons';
 import Button from '../../../../common/Button';
 import Input from '../../../../common/Inputs';
 
 class VideoTranslation extends Component {
   state = { selectedFile: null };
 
+  startRecord = () => {
+    const { player } = this.refs;
+    this.setState({ recording: true });
+    const { recording } = this.state;
+    if (!recording) {
```

Comment:
@HemaSAli I think my same comment above, about this.state.recording inside startRecord(), applies here too.

Reply:
@tomduggan85 I will edit it here too :+1: